packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1632,6 +1632,16 @@ export interface BootstrapOptions {
    */
   readonly dockerConfigJson?: string;
 
+  /**
+
+   * Overrides the IP address to use for DNS queries within the
+   * cluster.
+   *
+   * @default - 10.100.0.10 or 172.20.0.10 based on the IP
+   * address of the primary interface.
+   */
+  readonly dnsClusterIp?: string;
```

Comment:
This doesn't seem relevant to the rest of the PR - can we split it up?

Reply:
.. yep will do.  I was trying to sneak in support for this other bootstrap.sh flag at the same time because the per-change cost of contributing to cdk is so high :/  Will split it out..

Reply:
Thanks.

Also - interested to hear about the costly parts of contributing, we are constantly looking for ways to improve that. If you drop you're feedback here i'll make sure it gets to the right place :)

Reply:
> interested to hear about the costly parts of contributing

For example, just ^here is ~3 weeks of PR grooming (a rebase/reupload about every week) before finding out that I need to make a trivial change (break out a single option into its own PR) before this PR will be accepted.

It's not _hard_, but this isn't my day job.  So checking/polling multiple outstanding PRs over a lengthy time period to see if there was some bitrot, or a reviewer has responded and the auto-close timer-of-doom has started is actually time consuming in aggregate.

(Normal community development stuff.  None of that is new or unique to this project, so I hope that doesn't sound too negative.)

Reply:
I understand. I agree that slow turn-around cycles are hurtful for productivity, and create contribution fatigue. Thanks for the feedback :pray:

Reply:
Moved the dnsClusterIp parts to https://github.com/aws/aws-cdk/pull/13890
Note, the two PRs will trivially merge conflict in a test case, so one of them will need to be rebased and rereviewed.